FuelLabs/fuel-indexer
# enhancement: welcome wizard

### Description

This PR revamps `forc index welcome` to ask a few questions and create a manifest accordingly.

### Demo

#### Example 1
[![asciicast](https://asciinema.org/a/G4ttExKJ3kXWSb3UOfxAOMInl.svg)](https://asciinema.org/a/G4ttExKJ3kXWSb3UOfxAOMInl)

#### Example 2
[![asciicast](https://asciinema.org/a/hOioaqSsxOv0LbIHA2lFTPTsF.svg)](https://asciinema.org/a/hOioaqSsxOv0LbIHA2lFTPTsF)

### Testing steps
1. `cargo run -p forc-index -- welcome`
2. Read the prompts and answer the questions.
3. Ensure that the resultant project builds correctly.

### Notes
I'm down to change the name to something else. Additionally, perhaps we can augment `forc index new` with this functionality; for example, running `forc index new` with no args runs this series of prompts while including args would be considered "power user mode" and would require the usual args that are passed.
